src/routes/api/home/data.js

```diff
@@ -79,6 +79,104 @@ export const prod_desc = {
   `,
 };
 
+export const games = {
+  category: [
+    {
+      title: "DLC",
+      imgUrl: "https://cdn.mos.cms.futurecdn.net/7Cbpe8JbsPYc3ZfhzrP5ae-1200-80.jpg",
+    },
+  ],
+  list: [
+    {
```

Comment:
Your games data seems to contain much more data than you need.

Streamline it to only contain the data you need. So that when we build the backend api, we will use this to determine what needs to be sent down.